docs/testing.md

```diff
@@ -35,11 +35,14 @@ For faster feedback, you may want to set up [golanglint-ci's editor integration]
 E2e tests can be run in CI with the `/azp run e2e` command in your GitHub PR.
 
 E2e tests can also be run locally as follows:
-- Deploy or use an existing cosmos database
-- Run the RP
+- Make sure that you meet the requirements from [Prepare the database and run the rp](./deploy-development-rp.md) (do not create the database yet)
```

Comment:
AFAIK you can also use an existing cluster and just run make test-e2e if you let the CLUSTER variable point to it. Maybe we should also document this, because sometimes you might want to use an existing one.